Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -124,18 +124,15 @@ public struct AppStoreConnectClient {
     /// - Throws: An error of type `AppStoreConnectError` if the response indicates an error.
     public func fetchBuilds(
         for app: Application,
-        with query: BuildsQuery
+        with query: BuildsQuery?
     ) async throws -> [Build] {
+        let sort = query?.convertSort(query?.sort ?? .init( arrayLiteral: .hyphenVersion))
```

Comment:
It is incorrect. This parameter should not have a default value.

Reply:
Corrected.